package.json

```diff
@@ -48,6 +48,7 @@
     "chai": "3.5.0",
     "grunt": "0.4.5",
     "minimist": "1.2.0",
-    "mocha": "2.4.5"
+    "mocha": "2.4.5",
+    "webpack": "^1.13.3"
```

Comment:
Maybe we should lock this version down so it doesn't automatically update on us. Also 1.14.0 is stable, so we could lock into that version if we want.

So I would suggest: `"webpack": "1.14.0"` or `"webpack": "1.13.3"`